lib/diff/lcs/internals.rb

```diff
@@ -44,13 +44,12 @@ def lcs(a, b)
     b_finish = b.size - 1
     vector = []
 
-    # Prune off any common elements at the beginning...
+    # Collect any common elements at the beginning...
     while (a_start <= a_finish) and (b_start <= b_finish) and (a[a_start] == b[b_start])
       vector[a_start] = b_start
       a_start += 1
       b_start += 1
     end
-    b_start = a_start
```

Comment:
this doesn't seem necessary, since `b_start` will always equal `a_start` after the while block

Reply:
This is something where a larger corpus of test data would show this.

Reply:
Yeah please check, Here is my line of reasoning which leads to the conclusion:
The life cycle of a_start & b_start before the axed line:
- It's initialized to be 0 here:
https://github.com/halostatue/diff-lcs/blob/cd4b76d53d77fa596d72584d7531fdb811cd38bf/lib/diff/lcs/internals.rb#L42

- Then it's updated inside the loop here (both of them are updated):
https://github.com/halostatue/diff-lcs/blob/cd4b76d53d77fa596d72584d7531fdb811cd38bf/lib/diff/lcs/internals.rb#L50-L51

This means they must be equal when it comes to the axed line.